src/test/java/io/stargate/sgv2/jsonapi/service/operation/tables/WhereAnalyzerTest.java

```diff
@@ -572,6 +574,21 @@ public void comparison_api_filter_indexed_column() {
               .assertExceptionOnDurationColumns(cqlDatatypeColumn);
           continue;
         }
+
```

Comment:
nit pick: you could make this a  @ParameterizedTest and build the Fixtures in a method  and pass them into test. Also loop with three continues feels off, consider if/else